### PR TITLE
Added few test cases for profileView.spec.js and an element for Profi…

### PR DESCRIPTION
**Pull request by JuliaEde**
…le.page.js

### CODE REVIEW

File: test/ui/specs/user/profileView.spec.js [hunk 1]
```diff
@@ -9,19 +9,15 @@ describe('View Profile Page', () => {
...
         await LoginPage.login(process.env.USER_EMAIL, process.env.USER_PASSWORD);
     });
-
-    it('Should display Full name and Job Title', async () => {
+//https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2080765823&range=8:8
+    it('PP - 03: Should Display Full name', async () => {
         await PublicationsPage.navBar.openProfile();
         await ProfilePage.userFirstLastName.waitForDisplayed({timeout: 5000});
-
         const temp = await ProfilePage.userFirstLastName;
```
Comment (yuliahar) [inline test/ui/specs/user/profileView.spec.js:16]: Please change 16th and 17th lines to
```
        await expect(ProfilePage.userFirstLastName).toBeDisplayed();
```

File: test/ui/specs/user/profileView.spec.js [hunk 2]
```diff
@@ -33,7 +29,36 @@ describe('View Profile Page', () => {
         await ProfileEditPage.btnSave.click();
         await expect(ProfilePage.userInitials).toHaveText('MJ');
     });
-
+//https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2080765823&range=3:3
+    it('PP - 02: Should Verify that Profile page is not available for non-authorized user', async () => {
+        await PublicationsPage.navBar.openProfile();
+        const url = await browser.getUrl();
```
Comment (yuliahar) [inline test/ui/specs/user/profileView.spec.js:35]: Please try this
```
        const url = await browser.getUrl();
        await ProfilePage.navBar.logout();
        await browser.url(url);
        await browser.waitUntil(async () => {
            let pageUrl = await browser.getUrl();
            return pageUrl.indexOf('login') > -1
        }, 5000);
        const redirectedUrl = await browser.getUrl();
        await expect(redirectedUrl).toEqual('https://enduring.netlify.app/login');
```

File: test/ui/specs/user/profileView.spec.js [hunk 3]
```diff
@@ -33,7 +29,36 @@ describe('View Profile Page', () => {
...
+        await browser.url(url + '');
+        await browser.pause(4000);//doesn't work without pause (because it's redirecting
+        const redirectedUrl = await browser.getUrl();
+        await expect(redirectedUrl).toEqual('https://enduring.netlify.app/login');
+    });
+//https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2080765823&range=9:9
+    it('PP - 04: Should Display Job title', async () => {
+        await LoginPage.login(process.env.USER_EMAIL, process.env.USER_PASSWORD);
+        await PublicationsPage.navBar.openProfile();
+        await expect(ProfilePage.userJobTitle).toBeExisting();
```
Comment (yuliahar) [inline test/ui/specs/user/profileView.spec.js:46]: It's better to use .toBeDisplayed() and try to fail this test

File: test/ui/specs/user/profileView.spec.js [hunk 4]
```diff
@@ -33,7 +29,36 @@ describe('View Profile Page', () => {
...
+    });
+//https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2080765823&range=10:10
+    it('PP - 05: Should Display Email', async () => {
+        await PublicationsPage.navBar.openProfile();
+        await expect(ProfilePage.userEmail).toBeExisting();
```
Comment (yuliahar) [inline test/ui/specs/user/profileView.spec.js:51]: It's better to use .toBeDisplayed() and try to fail this test

File: test/ui/specs/user/profileView.spec.js [hunk 5]
```diff
@@ -9,19 +9,15 @@ describe('View Profile Page', () => {
     before('User Should Log In with Valid Credentials', async () => {
         await LoginPage.login(process.env.USER_EMAIL, process.env.USER_PASSWORD);
     });
-
-    it('Should display Full name and Job Title', async () => {
+//https://docs.google.com/spreadsheets/d/1Eg-BVQyju1YcKWL2vJYnRgLNliGOS7Vpn982QCOd1_s/edit#gid=2080765823&range=8:8
```
[the remark this answers is not in the document]
Comment (JuliaEde) [reply test/ui/specs/user/profileView.spec.js:12]: done, removed, thank you for the feedback!

File: test/ui/specs/user/profileView.spec.js (hunk 1 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileView.spec.js:16]: Thank you, changed!

File: test/ui/specs/user/profileView.spec.js (hunk 2 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileView.spec.js:35]: Replaced, thank you!

File: test/ui/specs/user/profileView.spec.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileView.spec.js:46]: I will try!

File: test/ui/specs/user/profileView.spec.js (hunk 4 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileView.spec.js]: I replaced with toBeDisplayed, but it didn't fail

**Pull request merged**